enos/enos-modules.hcl

```diff
@@ -1,17 +1,16 @@
 # Copyright (c) HashiCorp, Inc.
 # SPDX-License-Identifier: BUSL-1.1
 
-module "az_finder" {
-  source = "./modules/az_finder"
+module "aws_az_finder" {
```

Comment:
That `az` part of the name made me think "what does it have to with Azure" for a second 😅